core/email_api.php

```diff
@@ -1301,6 +1301,77 @@ function email_bug_reminder( $p_recipients, $p_bug_id, $p_message ) {
 }
 
 /**
+ * Send a notification to user or set of users that were mentioned in an issue
+ * or an issue note.
+ *
+ * @param integer       $p_bug_id     Issue for which the reminder is sent.
+ * @param array         $p_mention_user_ids User id or list of user ids array.
+ * @param string        $p_message    Optional message to add to the e-mail.
+ * @param array         $p_removed_mention_user_ids  The users that were removed due to lack of access.
+ * @return array List of users ids to whom the reminder e-mail was actually sent
+ */
+function email_user_mention( $p_bug_id, $p_mention_user_ids, $p_message, $p_removed_mention_user_ids = array() ) {
+	if( OFF == config_get( 'enable_email_notification' ) ) {
```

Comment:
Unless we in the code we specify to only look at all users/projects, then it can be done.  That doesn't mean we really think it is a good idea or that it will work right.  I use this explicit approach when checking if mentions are enabled.  We often should but don't use this format, maybe we should have a method that makes it easy, e.g. config_get_global_or_db() that just takes the config name and makes a call like the one below.
config_get( 'enable_email_notification', null, ALL_USERS, ALL_PROJECTS )